extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -394,6 +395,16 @@ UnremovableBeanBuildItem ensureJsonParserAvailable() {
                 "javax.json.bind.Jsonb");
     }
 
+    @BuildStep
+    void registerServiceBinding(Capabilities capabilities,
```

Comment:
For my curiosity is it binding or bindings.

The repo is plural: https://github.com/k8s-service-bindings/spec
The title is singular...

Reply:
I don't know to tell you the truth :)